src/main/java/net/revelc/code/formatter/xml/lib/XmlDocumentFormatter.java

```diff
@@ -13,6 +13,7 @@
  * 	   IBM Corporation - bug fixes
  * 	   Jose Montoya - Modified implementation outside Eclipse Platform
  * 	                - Add thread safety to TagReaderFactory
+ *     Saikiran Revuru - Modified copyNode method to delete blank lines
```

Comment:
The original author included this in the license header, but I don't think this is a best practice long-term. Most contributors aren't going to bother updating these headers every time they submit a modification. It probably makes more sense to add contributor information to the pom.xml file, so its in a central place. The existing one (Jose) can stay, though, just because it's already there.